contrib/carbonapi/fpm/carbon-user-systemd-reload.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+# based on https://github.com/leoleovich/grafsy/blob/master/packaging/postinst
+
+# Abort if any command returns an error value
+set -e
+
+USER=carbon
+GROUP=carbon
+CONF=/etc/carbonapi/carbonapi.yaml
+PIDFILE=/var/run/carbonapi/carbonapi.pid
+LOGFILE=/var/log/carbonapi/carbonapi.log
+
+# Following user part should be tested on both RPM and DEB systems
+if ! getent group "${GROUP}" > /dev/null 2>&1 ; then
+  groupadd --system "${GROUP}"
+fi
+GID=$(getent group "${GROUP}" | cut -d: -f 3)
+if ! id "${USER}" > /dev/null 2>&1 ; then
+  adduser --system --home /dev/null --no-create-home \
+    --gid "${GID}" --shell /bin/false \
+    "${USER}"
+fi
+
+# fix PID permissions
+if [ -f "${PIDFILE}" ]; then
+    chown "${USER}":"${GROUP}" "${PIDFILE}"
```

Comment:
Not sure if that's good idea. Theoretically, we should check ownership and permission of pid and logfile and change it only if not com patible with current version - but that make script quite complex....